### PR TITLE
app-admin/socklog: Bump to EAPI=6 Gentoo-Bug: 618574

### PR DESCRIPTION
**Pull request by charIes17**
Should resolve https://bugs.gentoo.org/618574

Package-Manager: Portage-2.3.10, Repoman-2.3.3

### CODE REVIEW

File: app-admin/socklog/socklog-2.1.0.ebuild [hunk 1]
```diff
@@ -16,10 +16,12 @@ IUSE="static"
...
 
+PATCHES=( "${FILESDIR}"/${PN}-2.1.0-headers.patch )
+
 S=${WORKDIR}/admin/${P}/src
 
 src_prepare() {
-	epatch "${FILESDIR}"/${PN}-2.1.0-headers.patch
+	default
 	use static && append-ldflags -static
 	echo "$(tc-getCC) ${CFLAGS} ${CPPFLAGS}" > conf-cc
```
Comment (monsieurp) [inline app-admin/socklog/socklog-2.1.0.ebuild]: missing `|| die`

File: app-admin/socklog/socklog-2.1.0.ebuild [hunk 2]
```diff
@@ -16,10 +16,12 @@ IUSE="static"
...
 	echo "$(tc-getCC) ${CFLAGS} ${LDFLAGS}" > conf-ld
```
Comment (monsieurp) [inline app-admin/socklog/socklog-2.1.0.ebuild]: Ditto.

Comment (charIes17) [reply app-admin/socklog/socklog-2.1.0.ebuild]: Thanks for reviewing. All changes done.

**Comment by monsieurp** [comment]
Also, an EAPI version upgrade requires a revision bump.

**Pull request closed without merging**